app/_src/deck/reference/deck_diff.md

```diff
@@ -36,7 +42,7 @@ and exit code 1 if an error occurs. (Default: `false`)
 :  Maximum number of concurrent operations. (Default: `10`)
 
 `--rbac-resources-only`
-:  sync only the RBAC resources (Kong Enterprise only). (Default: `false`)
+:  sync only the RBAC resources ({{site.ee_product_name}} only). (Default: `false`)
```

Comment:
```suggestion
:  sync only the RBAC resources **{{site.ee_product_name}} only**. (Default: `false`)
```
Not really anything to do with your PR but the parenthesis back to back looks weird to me.